run.py

```diff
@@ -485,6 +491,6 @@ def run_diffusion_processsing(**args):
                             for stage, stage_func in dwi_stage_dict.iteritems():
                                 if stage in args.stages:
                                     stage_func()
-                        except:
+                        except NameError:
                             print("You may have missing diffusion data in the positive phase encoding direction.")
```

Comment:
This tra/except statement still encompases a huge chunk of code that could produce "NameError" that does not necessarily mean that there is missing diffusion data in the positive phase encoding direction. This should be made more specific. Furthermore we should exit with a non zero code to indicate that something did not work correctly.